### ISSUE TITLE
 file:/// sheme bug: the extension is blinking 

### ISSUE THREAD

**Issue by filip26** [jusio/storage-area-explorer#15]

**Comment by kav2k** [comment]
I guess the same issue happens on `chrome-devtools://` scheme.

_Sup dev, I heard you like Dev Tools, so.._

Okay, okay.

1) Open an undocked Dev Tools window.
2) Switch to SAE tab.
3) Press Ctrl+Shift+I to open Dev Tools for the Dev Tools window.
4) Switch to SAE tab.
5) Enjoy the show.

**Comment by kav2k** [comment]
Oh wait, that's the same as #11

**Comment by jusio** [comment]
@kav2k Yep, most likely it is the same as #11. But if you want to inspect storage for devtools extension, you can always inspect background page.
